src/python/impactx/dashboard/jupyterApplication.py

```diff
@@ -0,0 +1,28 @@
+from .__main__ import application
+
+
+class JupyterMainApplication:
+    """
+    Creates specific components from
+    Trame Application for Jupyter Notebook.
+    """
+
+    def __init__(self):
+        self.ui = self.generate_ui()
+
+    def generate_ui(self):
+        return application()
+
+
+"""
+from dashboard.jupyterApplication import JupyterMainApplication
+
+# Create new application instance
+app = JupyterMainApplication()
+
+# Let's start the server by waiting for its UI to be ready
+await app.ui.ready
+
+# Put the UI into the resulting cell
+app.ui
+"""
```

Comment:
Is this a comment that we can remove? We have it in the docs now.
```suggestion
```

Reply:
Yup, we can remove it now since it is in the docs.

Reply:
Removed it in my last commit :)